test/navigation/ItemsNavigatorSpec.scala

```diff
@@ -35,9 +37,19 @@ class ItemsNavigatorSpec extends SpecBase {
 
       "must go from the Item Brand Name page" - {
 
+        "to the Commercial Description Page" in {
+          val userAnswers = emptyUserAnswers.set(ItemBrandNamePage(testIndex1), ItemBrandNameModel(hasBrandName = true, Some("brand")))
+
+
+          navigator.nextPage(ItemBrandNamePage(testIndex1), NormalMode, userAnswers) mustBe
+            itemsRoutes.CommercialDescriptionController.onPageLoad(testErn, testDraftId, Index(0), NormalMode)
```

Comment:
[Optional] `Index(0)` can be changed to `testIndex1`

Reply:
changed